src/main/java/org/fundacionjala/pivotal/common/commonMeth.java

```diff
@@ -0,0 +1,9 @@
+package org.fundacionjala.pivotal.common;
+
+/**
+ * Interface CommonMeth.
+ */
+public interface commonMeth {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this interface name to match the regular expression '^[A-Z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00114)